docs/website/blog/2023-11-01-dlt-dagster.md

```diff
@@ -0,0 +1,421 @@
+        pipeline = dlt.pipeline(
+        pipeline_name=self.pipeline_name, destination=self.destination, dataset_name=self.dataset_name
+        )
+
+        # run the pipeline with your parameters
+        load_info = pipeline.run(dlt_resource, table_name=table_name)
+
+        return load_info
+```
+
+- Define the asset, **`issues_pipeline`**, in **`assets/__init__.py`**. This asset uses the configurable resource to create a dlt pipeline and ingests data into BigQuery.
+
+```python
+from dagster import asset, get_dagster_logger
+from ..resources import DltResource
+from ..dlt import github_issues_resource
+
+@asset
+def issues_pipeline(pipeline: DltResource):
+
+    logger = get_dagster_logger()
+    results = pipeline.create_pipeline(github_issues_resource, table_name='github_issues')
+    logger.info(results)
+```
+
+The defined asset (**issues_pipeline**) takes as input the configurable resource (**DltResource**). In the asset, we use the configurable resource to create a dlt pipeline by using an instance of the configurable resource (**DltResource**) to call the `create_pipeline`  function. The `dlt.resource` (**github_issues_resource**) is passed to the `create_pipeline` function. The `create_pipeline` function normalizes the data and ingests it into BigQuery.
+
+### Step 5: Handle Schema Evolution
+
+`dlt` provides the feature of schema evolution that monitors changes in the defined table schema. Suppose GitHub adds a new column or changes a datatype of a column this small change can break pipelines and transformations. The schema evolution feature works amazingly well with dagster. 
+
+- Add the schema evolution code to the asset to make our pipelines more resilient to changes.
+
+```python
+from dagster import AssetExecutionContext
+@asset
+def issues_pipeline(context: AssetExecutionContext, pipeline: DltResource):
+...
+md_content=""
+    for package in result.load_packages:
+        for table_name, table in package.schema_update.items():
+            for column_name, column in table["columns"].items():
+                md_content= f"\tTable updated: {table_name}: Column changed: {column_name}: {column['data_type']}"
+
+    # Attach the Markdown content as metadata to the asset
+    context.add_output_metadata(metadata={"Updates": MetadataValue.md(md_content)})
+```
+
+### Step 6: Define Definitions
+
+- In the `__init.py__` under the **github_issues** folder add the definitions:
+
+```python
+all_assets = load_assets_from_modules([assets])
+simple_pipeline = define_asset_job(name="simple_pipeline", selection= ['issues_pipeline'])
+
+defs = Definitions(
+    assets=all_assets,
+    jobs=[simple_pipeline],
+    resources={
+        "pipeline": DltResource(
+            pipeline_name = "github_issues",
+            dataset_name = "dagster_github_issues",
+            destination = "bigquery",
+            table_name= "github_issues"
+        ),
+    }
+)
+```
+
+### Step 7: Run the Web Server and Materialize the asset
```

Comment:
is this a gpt style? :D capitalize words if it's really necessary, e.g. Materialize could be written as materialize, this is not a proper name